src/System.Text.Json/src/System/Text/Json/Writer/Utf8JsonWriter.WriteProperties.Bytes.cs

```diff
@@ -13,11 +13,8 @@ public sealed partial class Utf8JsonWriter
         /// <summary>
         /// Writes the pre-encoded property name and raw bytes value (as a base 64 encoded JSON string) as part of a name/value pair of a JSON object.
         /// </summary>
-        /// <param name="propertyName">The JSON encoded property name of the JSON object to be transcoded and written as UTF-8.</param>
-        /// <param name="bytes">The binary data to be written as a base 64 encoded JSON string as part of the name/value pair.</param>
-        /// <remarks>
-        /// The property name should already be escaped when the instance of <see cref="JsonEncodedText"/> was created.
-        /// </remarks>
+        /// <param name="propertyName">The JSON encoded name of the property to write.</param>
+        /// <param name="bytes">The 64 base encoded data to write.</param>
```

Comment:
Search/replace "64 base" => "Base 64"

Reply:
Fixed.

Reply:
When reviewing https://github.com/dotnet/dotnet-api-docs/pull/2737, I came across this change and had to do a double take.

I don't think this change makes sense. Do you recall what motivated this change, @safern?
```diff
- The binary data to be written as a base 64 encoded JSON string as part of the name/value pair.
+ The Base64-encoded data to write.
```

The bytes/data is **not** itself Base64-encoded. It is just binary data. It is **going to be** encoded to base64 before writing.

I preferred the original, but if we want to be less verbose, this works too:
```text
The binary data to be written as a Base64-encoded JSON string.
```
OR to match the sentence style to the proposed change:
```text
The binary data to write as Base64-encoded text.
```

Reply:
Sorry about that, I think then we should use:

```
The binary data to write as Base64-encoded text.
```

Do you want me to send a PR to master and release?

Reply:
> Do you want me to send a PR to master and release?

Yes, please. That would be awesome! Tyvm.